x-pack/plugins/session_view/public/components/tty_player/translations.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const PROCESS_DATA_LIMIT_EXCEEDED = i18n.translate(
+  'xpack.sessionView.processDataLimitExceeded',
+  {
+    defaultMessage: '  ⚠  Data limit reached for process',
+  }
+);
+
+export const PROCESS_DATA_LIMIT_EXCEEDED_2 = i18n.translate(
```

Comment:
what about `PROCESS_DATA_LIMIT_EXCEEDED_START` and `PROCESS_DATA_LIMIT_EXCEEDED_END` ?